src/utils/interpolate.hpp

```diff
@@ -0,0 +1,134 @@
+#ifndef INTERPOLATE_HPP_
```

Comment:
I can imagine this name `INTERPOLATE_HPP_` easily occurring in a downstream code. Our usual include guard format would be `UTILS_INTERPOLATE_HPP_`, which is maybe still a bit risky but definitely safer. 

Maybe we should expand all our include guards to have a `PARTHENON_` prefix...

Reply:
fixed